diff --git a/kernel/policy/allowlist.c b/kernel/policy/allowlist.c
index d2bf80a2..b1a9bf56 100644
--- a/kernel/policy/allowlist.c
+++ b/kernel/policy/allowlist.c
@@ -495,7 +495,7 @@ static void migrate_profile(u32 version, struct app_profile *profile)
         if (profile->allow_su) {
             domain = profile->rp_config.profile.selinux_domain;
             if (strncmp(domain, "u:r:su:s0", domain_len) == 0) {
-                strscpy_pad(domain, KSU_DEFAULT_SELINUX_DOMAIN, domain_len);
+                __strscpy_pad(domain, KSU_DEFAULT_SELINUX_DOMAIN, domain_len);
                 pr_info("migrated domain of profile: %s\n", profile->key);
             }
         }
@@ -634,4 +634,4 @@ void __exit ksu_allowlist_exit(void)
 		put_perm_data(np);
 	}
 	mutex_unlock(&allowlist_mutex);
-}
\ No newline at end of file
+}
